src/base/bittorrent/session.h

```diff
@@ -352,6 +360,8 @@ namespace BitTorrent
         void setPeerTurnoverCutoff(int val);
         int peerTurnoverInterval() const;
         void setPeerTurnoverInterval(int val);
+        DiskIOType diskIOType() const;
```

Comment:
These two declarations should be placed near `qint64 diskQueueSize() const;`.
And session.cpp needs adjustment too.